### PR TITLE
Grafana tutorials update

### PR DESCRIPTION
**Pull request by tonypowa**
This PR updates the steps to create alert rules in two tutorials 

- [X] The preview in Killercoda dev looks good to me

Source doc PR: https://github.com/grafana/grafana/pull/95134

### CODE REVIEW

**Comment by Jayclifford345** [comment]
@jdbaldry, is this a new way of generating links in the writer's tool kit? Should we modify these two regex replacements for the time being?

**Comment by jdbaldry** [comment]
Ah yeah, it's https://grafana.com/docs/writers-toolkit/write/links/#link-from-source-content-thats-reused-as-multiple-pages. It's for content that's reused and has different link destinations depending on the page's location.

I think regex replacements is definitely the quickest fix until we can sit down to think about how to handle this Hugo feature in the transformer.

**Comment by jdbaldry** [comment]
Before we put the regex replacements in, it's worth double checking that we actually need to use `ref` URIs in this document. Is  this page content that's in both the Grafana OSS Alerting docs and also in Grafana Cloud Alerting docs?

**Comment by tonypowa** [comment]
i actually think we can remove the ref and move on because the tutorials live in https://grafana.com/tutorials/ , which is unrelated to /docs/ . However, we are discussing with the docs team the possibility to move the tutorials into the docs in the future. But that's a tomorrow's problem 😁 

i'm removing the refs..

**Comment by Jayclifford345** [comment]
Ah legend thanks @tonypowa! I will create an issue in killercoda if it isn't already there. Will have a scan now :)

**Comment by tonypowa** [comment]
✔️  done @Jayclifford345  🦖 
also, thank you @jdbaldry  !

File: grafana/alerting-get-started/preprocessed.md [hunk 1]
```diff
@@ -195,18 +194,15 @@ In this section, we define queries, expressions (used to manipulate the data), a
...
-   - Change the **Input** to **'A'** to select the data source.
-   - Enter `0` as the threshold value. This is the value above which the alert rule should trigger.
-
-1. Click **Preview** to run the queries.
+1. Click **Preview alert rule condition** to run the query.
 
    It should return a single sample with the value 1 at the current timestamp. And, since `1` is above `0`, the alert condition has been met, and the alert rule state is `Firing`.
 
-   {{< figure src="/media/docs/alerting/alerting-always-firing-alert.png" max-width="1200px" caption="A preview of a firing alert" >}}
+   {{< figure src="/media/docs/alerting/firing-alert-preview.png" max-width="1200px" caption="A preview of a firing alert" >}}
```
Comment (Jayclifford345) [inline grafana/alerting-get-started/preprocessed.md:205]: Aaah are you running the transformer manually for this update @tonypowa. I believe @jdbaldry fixed this issue in the transformer. Would you mind pulling the latest version of the staging branch and rebuilding the transformer? 

If you would like you could also just let the automation tool pick these changes up if they are merged into the source repos

Comment (tonypowa) [reply grafana/alerting-get-started/preprocessed.md:205]: >If you would like you could also just let the automation tool pick these changes up if they are merged into the source repos

ahh i already merged the changes into the source doc! 😓  I'm okay waiting for the continuous agent to kick off

Comment (tonypowa) [reply grafana/alerting-get-started/preprocessed.md:205]: @Jayclifford345

Comment (jdbaldry) [reply grafana/alerting-get-started/preprocessed.md:205]: https://github.com/grafana/killercoda/pull/148/ I think

Comment (Jayclifford345) [reply grafana/alerting-get-started/preprocessed.md:205]: I think we still have a problem with one of the images but I am merging to not block the pipeline. Its not a deal breaker and it was there from our initial commit

Comment (jdbaldry) [reply grafana/alerting-get-started/preprocessed.md:205]: What's up with the image? Anything I can do to fix it?

Comment (Jayclifford345) [reply grafana/alerting-get-started/preprocessed.md:205]: @jdbaldry sorry false alarm my dude! I was reading the preprocessed markdown and thought the transformer was missing the images but all good in the actual stages

Comment (jdbaldry) [reply grafana/alerting-get-started/preprocessed.md:205]: No worries at all :)

**Pull request closed without merging**